src/index.html

```diff
@@ -16,7 +28,95 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="menu">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+
+      <nav>
+        <ul class="menu_box">
+          <li class="menu_item">
+            <a
+              class="menu_link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="menu_item">
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="menu_item">
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="menu_item">
+            <a
+              class="menu_link"
+              data-qa="hover"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="menu_item">
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="menu_item">
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="menu_item">
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="menu_item">
+            <a
+              class="menu_link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. For example, add an empty line between the closing </li> and the next <li> for each item in the menu.